book/security.md

```diff
@@ -142,15 +142,14 @@ You'll also need to modify the argument lists for `add_entry` and
 each user accessing the guest book, we can now build a login system.
 
 ::: {.further}
-The [patent][patent] for cookies says there is "no compelling reason"
```

Comment:
I think this might be the wrong patent. Also, I don't like software patents.

Reply:
Agreed, but it's still of historic interest, and the only available textual reference I could find on the name.